### DISCUSSION TITLE
autoortho does not start!

### DISCUSSION THREAD

**Discussion by Alexfx82** [kubilus1/autoortho#461] [General]
I have a very strange problem! Just yesterday I launched autoortho and flew calmly ! But today the program does not start. I reinstalled Python and dokan, but still the program does not start! What's the matter?

**Comment by Alexfx82** [comment]
Is it possible that the reason is that I have installed the latest version of python?

**Comment by Alexfx82** [comment]
Can anyone help?

**Comment by bld481** [comment]
Same problem for me. Worked fine one day, but now not at all no matter what I install, reinstall, remove, or exempt from anti-virus.

**Reply by Alexfx82** [reply]
Thanks for the answer, looks like I'm not alone! I hope the developer will hear us

**Comment by Alexfx82** [comment]
Windows 11! I installed and deleted Dokan, installed and deleted Winfsp! The program won't start anyway! I can't figure out what's the matter, it used to open

**Reply by hdair** [reply]
currently in windows 10, having the same issue, hope the developer address this

**Comment by VirtualGAa** [comment]
Add me to the list. When I click the executable in %appsdata% nothing, I mean absolutely nothing happens. Task manager shows no sign of executing any commands.

AutoOrtho Version 0.7.1 on top of Win 11 home 10.0.22621  
Running:  Windows Defender and Malwarebytes | Dokan Lib 2.1.0.1000 Bundle | Python 3.12.0 with versions 3.11, 38, & 37 in the path 

fwiw - I can offer a video outlining a clean install, if requested by the developer.

**Comment by Alexfx82** [comment]
Everything is the same for me as for you! Dear developer! Please pay attention to our problem! Tnx

**Comment by bld481** [comment]
It seems as if there will be no answers for this. That's too bad X-Plane 12 is not the same without it.

**Comment by bld481** [comment]
I thought I would try installing it again after posting that comment, and when I clicked on the shortcut it installed, nothing happened. Then I thought, what if I launch an administrator command prompt and grab the path to the exe from the install folder? Success!! I do not know if it is the technically correct thing to do, but I changed the path of its shortcut to the exe path I got from the install folder, and it works now.

**Reply by bld481** [reply]
I think that shortcut may have pointed to the icon, not the exe.

**Reply by Alexfx82** [reply]
You are lucky! BUT it doesn't work for me, I run the executable file from the root of the /autoortho folder and nothing happens

**Reply by bld481** [reply]
I do not know why running the exe from the command prompt seems to make it work for me, even though that doesn't make any sense because I, like you, have tried running the exe by double-clicking in the root folder. All I know is that the shortcut pointed to the .ico file, and after I ran the exe from an elevated command prompt, it worked. I hope you find a solution.

**Reply by Alexfx82** [reply]
I can't run it from the command line, I think it's something with python, because the system says that: Windows PowerShell does not load commands from the current location

**Comment by Alexfx82** [comment]
Eureka!!!! I found a mistake! Delete the \user\\.autoortho file and run the program

**Reply by VirtualGAa** [reply]
I've been following this thread closely. Where is the \user\.autoortho file? 
I'm launching the program directly from the programs folder (c:\programs\autoortho..).

**Reply by Alexfx82** [reply]
For example:   c:\users\alex\\.autoortho
